draft-ietf-teep-architecture.md

```diff
@@ -1157,8 +1172,13 @@ to determine that the TAM is trustworthy.
 ## Malicious TA Removal
 
 It is possible that a rogue developer distributes a malicious Untrusted 
-Application and intends to get a malicious TA installed. It's the responsibility
-of the TAM to not install malicious trusted apps in the first place. The TEEP
+Application and intends to get a malicious TA installed. Such a TA
+might be able to hide from malware detection the REE, or access trusted
+resources within the TEE (but could not access other TEEs, or access other
```

Comment:
to hide from malware detection the REE, -->
to escape from malware detection by the REE